custom_components/healthchecksio/config_flow.py

```
@@ -109,6 +110,77 @@
+            ): selector.BooleanSelector(selector.BooleanSelectorConfig()),
+            vol.Optional(
+                CONF_SELF_HOSTED,
+                default=user_input.get(
+                    CONF_SELF_HOSTED, fallback.get(CONF_SELF_HOSTED, DEFAULT_SELF_HOSTED)
+                ),
+            ): selector.BooleanSelector(selector.BooleanSelectorConfig()),
+        }
+    )
+
+
+def _build_self_hosted_schema(
```

Pylint (3.3.7)

[warning] Dangerous default value {} as argument

(W0102)